src/_data/sites/statox.json

```diff
@@ -2,5 +2,6 @@
     "url": "https://www.statox.fr/",
     "name": "The stuff I do",
     "description": "The personal blog of a backend software engineer",
-    "source_url": "https://github.com/statox/blog/"
+    "source_url": "https://github.com/statox/blog/",
+    "authoredBy" : ["statox"]
```

Comment:
Ah I might  be missing something but as you can see on the production website for now statox.fr is not listed but locally if I had `twitter` or `authoredBy` it shows up. 

I can dig the code a bit more to clarify the behavior.

Reply:
@zachleat I should ~probably~ definitely know this by now, but&hellip;

Is `twitter` a required field for sites in src/_data/sites/*.json? Or does the 11ty.dev site need to be redeployed?

Reply:
Looks like we have about 63 sites without `"twitter"` accounts, per:

```sh
git grep -L "twitter" src/_data/sites | wc -l # 63
```

But even though statebox.json doesn't have a "twitter" account, it does still show up in the latest leaderboards: https://www.11ty.dev/speedlify/#site-7886e9ad

Reply:
I think `statox.fr` does not appear on the leaderboards https://www.11ty.dev/speedlify/ because the leaderboards needs a redeploy as the blog post I linked earlier explains. I think this will happen eventually so there is no issue here.

However I think there is an issue with the author page https://www.11ty.dev/authors/ : 

For example take `statox.fr`, `yoghaswara.json`, `vagabondians.json` and `troyv.json` some of them appear in the leaderboards (probably because they were added before the last deploy) but none of them appears in the author page.

And this happens because the author page is built using the `topAuthor` filter which doesn't include in its results the sites which don't have a `twitter` or `authoredBy` property.

So I guess my question is: Is it expected that the sites without a linked twitter account are not listed in the author page ?